packages/data-table-server/src/dataTableService/services/AnalyticsDataTableService.ts

```diff
@@ -7,17 +7,20 @@ import { AccessPolicy } from '@tupaia/access-policy';
 import { TupaiaApiClient } from '@tupaia/api-client';
 import { Aggregator } from '@tupaia/aggregator';
 import { DataBroker } from '@tupaia/data-broker';
-import { EARLIEST_DATA_DATE_STRING, yup } from '@tupaia/utils';
+import { getDefaultPeriod, convertPeriodStringToDateRange, yup } from '@tupaia/utils';
 import { DataTableService } from '../DataTableService';
 import { orderParametersByName } from '../utils';
 import { mapProjectEntitiesToCountries } from './utils';
 
+const getDefaultStartDate = () => new Date(convertPeriodStringToDateRange(getDefaultPeriod())[0]);
+const getDefaultEndDate = () => new Date(convertPeriodStringToDateRange(getDefaultPeriod())[1]);
```

Comment:
Might wanna move to a shared util across `AnalyticsDataTableService` and `EventsDataTableService`? But no biggie either way really